src/main/java/org/baeldung/web/controller/UsersController.java

```diff
@@ -0,0 +1,24 @@
+package org.baeldung.web.controller;
+
+import java.util.Locale;
+
+import org.baeldung.security.LoggedUsers;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+
+@Controller
+public class UsersController {
+
+	@RequestMapping(value = "/loggedUsers", method = RequestMethod.GET)
+	public String getLoggedUsers(final Locale locale, final Model model) {
+		model.addAttribute("users", LoggedUsers.getInstance().getUsers());
+		System.out.println(LoggedUsers.getInstance().getUsers().size());
+		for (String user:LoggedUsers.getInstance().getUsers()){
+			System.out.println("logged in user:"+user);
+		}
+		return "redirect:/users.html";
```

Comment:
Please remove System.out.println statements.
You can revisit https://docs.google.com/document/d/1L-TjM1cP8biT6mIWvH5fYaFHtWyZytzQoqrJ92WqK1A/pub for more information on Logic and Formatting